doc/onboarding.md

```diff
@@ -116,39 +116,34 @@ onboarding session.
       * The remaining elements on the form are typically unchanged with the exception of `POST_STATUS_TO_PR`. Check that if you want a CI status indicator to be automatically inserted into the PR.
 
 
-## process for getting code in
+## Landing PRs: Overview
 
-  * the collaborator guide is a great resource: https://github.com/nodejs/node/blob/master/COLLABORATOR_GUIDE.md#technical-howto
+  * The [Collaborator Guide](https://github.com/nodejs/node/blob/master/COLLABORATOR_GUIDE.md#technical-howto) is a great resource.
 
 
-  * no one (including TSC or CTC members) pushes directly to master without review
-    * an exception is made for release commits only
+  * No one (including TSC or CTC members) pushes directly to master without review.
+    * An exception is made for release commits only.
 
 
-  * one "LGTM" is usually sufficient, except for semver-major changes
-    * the more the better
-    * semver-major (breaking) changes must be reviewed in some form by the CTC
+  * One "LGTM" is usually sufficient, except for semver-major changes.
+    * The more the better.
+    * Breaking changes must be LGTM'ed by at least two CTC members.
 
 
-  * be sure to wait before merging non-trivial changes
-    * 48 hours for non-trivial changes, and 72 hours on weekends.
+  * Wait before merging non-trivial changes.
+    * 48 hours during the week and 72 hours on weekends.
 
 
-  * **make sure to run the PR through CI before merging!** (Except for documentation PRs)
+  * **Run the PR through CI before merging!**
+    * An exception can be made for documentation-only PRs.
 
-
-  * once code is ready to go in:
-    * [**See "Landing PRs"**](#landing-prs) below
-
-
-  * what if something goes wrong?
-    * ping a CTC member
+  * What if something goes wrong?
+    * Ping a CTC member.
     * `#node-dev` on freenode
-    * force-pushing to fix things after is allowed for ~10 minutes, be sure to notify people in IRC if you need to do this, but avoid it
-    * Info on PRs that don't like to apply found under [**"If `git am` fails"**](./onboarding-extras.md#if-git-am-fails).
+    * Force-pushing to fix things after is allowed for ~10 minutes. Avoid it if you can. Post to `#node-dev` (IRC) if you need to force push.
```

Comment:
I think `--force-with-lease` with lease is worth mentioning and will add it.

Happy to make the last part its own sub-bullet. I don't have a strong opinion either way.